academic_observatory_workflows/seed/dataset_info.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2020 Curtin University
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# Author: Tuan Chien
+
+
+from collections import OrderedDict
+
+from academic_observatory_workflows.api_type_ids import DatasetTypeId
+from observatory.api.client.api.observatory_api import ObservatoryApi
+from observatory.api.client.model.dataset import Dataset
+from observatory.api.utils import get_workflows, get_dataset_type
+
+
+def get_dataset_info(api: ObservatoryApi):
+    workflows = get_workflows(api=api)
+
+    dataset_info = OrderedDict()
+    dataset_info["Crossref Events Dataset"] = Dataset(
+        name="Crossref Events Dataset",
+        service="google",
+        address="academic-observatory.crossref.crossref_events",
+        workflow=workflows["Crossref Events Telescope"],
+        dataset_type=get_dataset_type(api=api, type_id=DatasetTypeId.crossref_events),
```

Comment:
Using constants defined in `academic_observatory_workflows.api_type_ids` to prevent mistakes.